### ISSUE TITLE
CVE-2024-36361

### ISSUE THREAD

**Issue by diegocr** [kucherenko/jscpd#629]
https://cvefeed.io/vuln/detail/CVE-2024-36361

```db
$ npm audit

pug  *
Severity: high
Pug allows JavaScript code execution if an application accepts untrusted input - https://github.com/advisories/GHSA-3965-hpx2-q597
fix available via `npm audit fix --force`
Will install jscpd@3.2.1, which is a breaking change
node_modules/pug
  @jscpd/finder  *
  Depends on vulnerable versions of pug
  node_modules/@jscpd/finder
    jscpd  >=3.3.0-alpha.2
    Depends on vulnerable versions of @jscpd/finder
    Depends on vulnerable versions of @jscpd/html-reporter
    node_modules/jscpd
  @jscpd/html-reporter  >=3.5.6
  Depends on vulnerable versions of @jscpd/finder
  Depends on vulnerable versions of pug
  node_modules/@jscpd/html-reporter

4 high severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force


$ npm ls pug
@webclient
└─┬ jscpd@3.5.10
  ├─┬ @jscpd/finder@3.5.10
  │ └── pug@3.0.2
  └─┬ @jscpd/html-reporter@3.5.10
    └── pug@3.0.2 deduped
```

**Comment by kucherenko** [comment]
fixed in jscpd@4.0.1

**Comment by diegocr** [comment]
Thank you!

(nice amount of changes there :))
